scripts/nvqc_launch.sh

```diff
@@ -0,0 +1,39 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+cat /opt/nvidia/cudaq/build_info.txt
```

Comment:
Most of this script is moving and unquoting commands from docker/release/cudaq.nvqc.Dockerfile into this script. I will highlight the one important change below.